theme/wds-headless-theme/inc/tgm/tgm.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Theme functionality.
+ *
+ * @author WebDevStudios
+ * @package wds-headless-theme
+ * @since 1.0
+ */
+
+// Require primary TGM class.
+require_once 'class-tgm-plugin-activation.php';
+
+/**
+ * List of required plugins.
+ *
+ * @see http://tgmpluginactivation.com/configuration/
+ * @author WebDevStudios
+ * @since 1.0
+ */
+function wds_register_required_plugins() {
+
+	$plugins = [
+
+		// Plugins from the WordPress Plugin Repository.
```

Comment:
We should probably add the following here as well:

```
// Plugins only available on Github.
[
	'name'     => 'WDS Headless Core',
	'slug'     => 'wds-headless-core',
	'source'   => 'https://github.com/WebDevStudios/nextjs-wordpress-starter',
	'source'   => 'https://nextjs.wpengine.com/downloads/wds-headless-core.zip',
	'required' => true,
],
```

Reply:
👍🏻